test/jdk/javax/accessibility/manual/ComboBoxDemo.html

```diff
@@ -44,25 +44,24 @@
         <td>ComboBox Demo</td>
         <td>
             <ol>
-                <li> Tab until ComboBox icon<img src="./resource/cmb.png"> has focus. Press 'Space' to choose.</li>
+                <li>Tab until ComboBox icon<img src="./resource/cmb.png"> has focus. Press 'Space' to choose.</li>
                 <li>Tab until the "ComboBox Demo" tab has focus. Press 'space'. Press 'tab'.</li>
-                <li> Use Tab and Shift-Tab to move between the four ComboBox widgets.
+                <li>Use Tab and Shift-Tab to move between the four ComboBox widgets.
                 <li>Use the space and down arrow keys to bring up the drop-down list.</li>
-                <li> Use the up and down arrows to navigate up and down the list.</li>
-                <li>Use the 'space' key to make a the selection.</li>
-                <li> Repeat 4,5 but hit Esc key to cancel the drop-down.</li>
+                <li>Use the up and down arrows to navigate up and down the list.</li>
+                <li>Use the 'Space' key to make the selection.</li>
+                <li>Repeat 4,5 but hit Esc key to cancel the drop-down.</li>
             </ol>
         </td>
-        </td>
     </tr>
     <tr>
         <td style="Width:100%;" colspan="3"><b>Expected Result</b></td>
     </tr>
     <tr>
         <td style="Width:100%;" colspan="3">
             <ol>
-                <li>Verify that space and down arrow bring up the drop-down list.</li>
-                <img src="./resource/list.png">
+                <li>Verify that space and down arrow bring up the drop-down list.<br>

```

Comment:
Is this HTML refactoring needed as part of this PR?

Reply:
The `<img>` element is placed after `</li>` and before the next `<li>`. In HTML `<ol>` doesn't allow anything but `<li>` elements. I placed the image to where it was supposed to be.

I decided it was easier and more efficient to fix it as part of this change rather than submit a new bug and create a new pull request afterwards.

Reply:
In fact, you're right: I looked at the other HTML files near this one, and some of them also contain similar markup errors. It'll be wiser to address them separately.

I submitted [JDK-8284958](https://bugs.openjdk.java.net/browse/JDK-8284958) and reverted all changes to `ComboBoxDemo.html` but ‘a the’ to ‘the’.